src/store/RootStore/hooks/useQueryParamsStoreInit.ts

```diff
@@ -0,0 +1,8 @@
+import * as Router from "react-router-dom";
```

Comment:
так как тут используешь только useLocation, можно только его и импортировать
```suggestion
import { useLocation } from "react-router-dom";
```